### ISSUE TITLE
[Bighorn RC2] Support overview broken

### ISSUE THREAD

**Issue by JeffreyDevloo** [openvstorage/framework#2006]
### Problem description
The package information and heartbeat data did not get displayed on the page

```
Mar 21 10:21:48 LC1SRV0008 celery[8683]: 2018-03-21 10:21:48 55100 +0100 - LC1SRV0008 - 8683/139926761338624 - celery/log.py - log - 98 - ERROR - Task ovs.storagerouter.get_support_metadata[7f299cf8-5ad4-4cb3-8357-68c9c341440c] raised unexpected: EncodeError(RepresenterError("cannot represent an object: OrderedDict([('ovs-albaproxy_myvpool01_0.service', 'loaded active running ALBA proxy'), ('ovs-albaproxy_myvpool01_1.service', 'loaded active running ALBA proxy'), ('ovs-dtl_myvpool01.service', 'loaded active running Open vStorage distributed transaction log'), ('ovs-scheduled-tasks.service', 'loaded active running Open vStorage scheduled tasks'), ('ovs-support-agent.service', 'loaded active running Open vStorage support agent'), ('ovs-volumedriver_myvpool01.service', 'loaded active running Open vStorage StorageDriver'), ('ovs-volumerouter-consumer.service', 'loaded active running Open vStorage volumerouter consumer'), ('ovs-watcher-config.service', 'loaded active running Watcher for the configuration cluster'), ('ovs-watcher-framework.service', 'loaded active running Watcher for Open vStorage master services'), ('ovs-watcher-volumedriver.service', 'loaded active running Watcher for Open vStorage master services'), ('ovs-webapp-api.service', 'loaded active running Open vStorage webapp API'), ('ovs-workers.service', 'loaded active running Open vStorage workers')])",),)
Mar 21 10:21:48 LC1SRV0008 celery[8683]: Traceback (most recent call last):
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 283, in trace_task
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     uuid, retval, SUCCESS, request=task_request,
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/celery/backends/base.py", line 271, in store_result
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     request=request, **kwargs)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/celery/backends/base.py", line 505, in _store_result
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     self.set(self.get_key_for_task(task_id), self.encode(meta))
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/celery/backends/base.py", line 195, in encode
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     _, _, payload = dumps(data, serializer=self.serializer)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/kombu/serialization.py", line 164, in dumps
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     payload = encoder(data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/contextlib.py", line 35, in __exit__
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     self.gen.throw(type, value, traceback)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/kombu/serialization.py", line 59, in _reraise_errors
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     reraise(wrapper, wrapper(exc), sys.exc_info()[2])
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/kombu/serialization.py", line 55, in _reraise_errors
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     yield
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/kombu/serialization.py", line 164, in dumps
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     payload = encoder(data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/__init__.py", line 218, in safe_dump
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     return dump_all([data], stream, Dumper=SafeDumper, **kwds)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/__init__.py", line 190, in dump_all
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     dumper.represent(data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 28, in represent
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     node = self.represent_data(data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 57, in represent_data
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     node = self.yaml_representers[data_types[0]](self, data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 223, in represent_dict
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     return self.represent_mapping(u'tag:yaml.org,2002:map', data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 123, in represent_mapping
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     node_value = self.represent_data(item_value)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 57, in represent_data
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     node = self.yaml_representers[data_types[0]](self, data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 223, in represent_dict
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     return self.represent_mapping(u'tag:yaml.org,2002:map', data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 123, in represent_mapping
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     node_value = self.represent_data(item_value)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 57, in represent_data
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     node = self.yaml_representers[data_types[0]](self, data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 223, in represent_dict
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     return self.represent_mapping(u'tag:yaml.org,2002:map', data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 123, in represent_mapping
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     node_value = self.represent_data(item_value)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 67, in represent_data
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     node = self.yaml_representers[None](self, data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]:   File "/usr/lib/python2.7/dist-packages/yaml/representer.py", line 247, in represent_undefined
Mar 21 10:21:48 LC1SRV0008 celery[8683]:     raise RepresenterError("cannot represent an object: %s" % data)
Mar 21 10:21:48 LC1SRV0008 celery[8683]: EncodeError: cannot represent an object: OrderedDict([('ovs-albaproxy_myvpool01_0.service', 'loaded active running ALBA proxy'), ('ovs-albaproxy_myvpool01_1.service', 'loaded active running ALBA proxy'), ('ovs-dtl_myvpool01.service', 'loaded active running Open vStorage distributed transaction log'), ('ovs-scheduled-tasks.service', 'loaded active running Open vStorage scheduled tasks'), ('ovs-support-agent.service', 'loaded active running Open vStorage support agent'), ('ovs-volumedriver_myvpool01.service', 'loaded active running Open vStorage StorageDriver'), ('ovs-volumerouter-consumer.service', 'loaded active running Open vStorage volumerouter consumer'), ('ovs-watcher-config.service', 'loaded active running Watcher for the configuration cluster'), ('ovs-watcher-framework.service', 'loaded active running Watcher for Open vStorage master services'), ('ovs-watcher-volumedriver.service', 'loaded active running Watcher for Open vStorage master services'), ('ovs-webapp-api.service', 'loaded active running Open vStorage webapp API'), ('ovs-workers.service', 'loaded active running Open vStorage workers')])
```

The Framework uses OrderedDicts for sorting purposes when retrieving packages/other info to use them further down the line (like in the update)
YAML can't encode the OrderedDict as it is not derived from the python dict in 2.7 (no issue in python 3.6)

**Comment by wimpers** [comment]
Can this be moved to code complete or is there anything else todo?

**Comment by JeffreyDevloo** [comment]
Fixed by #2008 
Released in https://github.com/openvstorage/framework/releases/tag/2.11.5
